1-exercises/C-array-every/exercise.js

```diff
@@ -5,7 +5,11 @@
 let students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 let group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-let groupIsOnlyStudents; // complete this statement
+function isStudents(student) {
+  return student === students;
+}
```

Comment:
This function isn't exactly doing what we expect it to do. If you remove the item "Daniel" from the group array and run the function, it breaks.  You just need to replace the === operator with an Array method that checks to see if a value is **included** among the array in doing the check on rather than checking whether they are equal.  You're very close.

Reply:
I fixed it. Thanks for letting me know.